### DISCUSSION TITLE
it stop working all of a sudden

### DISCUSSION THREAD

**Discussion by Syeech** [Daksh777/SpotifyNoPremium#90] [Q&A]
Idk what's going on, it's been happening for a while now, i've reinstall it, update it, i tried evything, but my spotify often just stop working, all of the songs just got greyed out, even after closing it and opening it up again. can you help me with this pls? thank you very much.

![image](https://user-images.githubusercontent.com/112376772/187143223-aa3f55e7-3f9c-4614-a338-6a9f7238f9ad.png)

**Comment by Daksh777** [comment]
Does this also happen on the stock Spotify app (i.e, the unmodified version)?

**Reply by Syeech** [reply]
uh yeah it's still greyed out after i restore it with the spicetify restore commands.. haven't tried if i uninstalled and reinstall it again..

![image](https://user-images.githubusercontent.com/112376772/187606874-9b80661a-5c09-4f70-b114-04b7b18b5de8.png)

**Reply by Daksh777** [reply]
Seems like a problem with Spotify itself and not my theme.
Maybe try reinstalling it and see if that works?

**Reply by Syeech** [reply]
ok, so i've tried reinstalling it, and now it runs ok, it hasn't stopped working yet, it's been an hour or so. 
so what should i do? i really like the no ads feature and have been using it for a long time tbh, but for a few months now it's been like that, greyed out songs, and straight up not working. is there anything i can do? thanks for your responds!

**Reply by Daksh777** [reply]
I can't think of a reason my theme would block songs from playing. 
Do you happen to have any private DNS or a VPN or something similar? If so, try to disable it and see if it works.

**Reply by Syeech** [reply]
yeah i dont think i have those on, idk what's wrong with it, guess i'll just have to live with it ig lol
thank you for your help tho, sorry i reply very late all the time,
have a great day! cool mod btw!

**Reply by Daksh777** [reply]
Do let me know if you figure out any solution so it can help others. Other than that, if you want, we can check a bunch of things and try some solutions to see if it works. You can DM me either on Telegram (https://telegram.me/Daksh777) or Discord (Lars24#9047)
